docs-src/zdm-core/modules/migrate/pages/change-read-routing.adoc

```diff
@@ -7,7 +7,7 @@ This topic explains how you can configure the {zdm-proxy} to route all reads to
 
 Illustrated view of this phase:
 
-image::{imagesprefix}migration-phase4ra.png[Phase 4 diagram shows read routing on ZDM Proxy was switched to Target.]
+image::{imagesprefix}migration-phase4ra9.png[Phase 4 diagram shows read routing on ZDM Proxy was switched to Target.]
```

Comment:
```suggestion
image::{imagesprefix}migration-phase4ra9.png["Phase 4 diagram shows read routing on ZDM Proxy was switched to Target."]
```

Reply:
Quotes in place from an interim commit today (thanks). Didn't do image:: as discussed. Latest is on coppi.